lib/sentry/rescue_evss_errors.rb

```diff
@@ -9,7 +9,7 @@ module RescueEVSSErrors
     def rescue_evss_errors(keys)
       yield
     rescue EVSS::ErrorMiddleware::EVSSError => e
-      if e.message&.match keys
```

Comment:
Was thinking about this and it was bothering me.  I don't know if it would ever happen, but if a "messages" text ever referenced a different key, we would get a false positive here.  Therefore, I  changed `EVSSError` to accept a `details` parameter so we can specifically match on the `key` in `RescueEVSSErrors`